Documentation/BasicPrerequisites.md

```diff
@@ -0,0 +1,15 @@
+# Basic Prerequisites for WheelMUD
+
+## Supported Operating Systems
+Most recent Windows environments should be able to run WheelMUD. The following have been explicitly reported as working without any code changes:
+* Windows XP (at least with SP2 and SP3)
+* Windows Vista (including SP1 and SP2)
+* Windows 7
+* Windows 2003 Server (including R2)
+* Windows 2008 Server (including R2)
+Others (such as Windows 8) should work; let us know how it goes in the [muds.gamedev.com](http://muds.gamedev.com) forums if you try another operating system.
+
+## Supported Development Environments
+Currently we are focusing on supporting one solution file, for use with Visual Studio 2013.  (The free, full-featured Visual Studio Community 2013 edition, is a good way to go.)
+The solution will likely continue working in paid VS2010 editions as well (although not supported).
```

Comment:
I'm using visual studio 2013 ultimate and it uses the same profile as professional so these are supported by simple use of my own process.